src/test/java/net/bhardy/bizzo/billing/PolicyBuilderTest.java

```diff
@@ -37,201 +39,339 @@
  *
  */
 public class PolicyBuilderTest {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)